README.md

```diff
@@ -33,10 +34,10 @@ The light weight containerized version of the HPE OneView SDK for Terraform is a
 
 ```bash
 # Download and store a local copy of oneview-sdk-for-terraform and use it as a Docker Image.
-$ docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-terraform:v1.6.0-OV5.5
+$ docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-terraform:v1.7.0-0.11-OV5.6
 # Run docker commands below given, which  will in turn create a sh session 
 # where you can create files, issue commands and execute the examples.
-$ docker run -it docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-terraform:v1.6.0-OV5.5 /bin/sh
+$ docker run -it docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-terraform:v1.7.0-0.11-OV5.6 /bin/sh
```

Comment:
same tag is v1.7.0-11

Reply:
tag v1.7.0-0.11 was making more sense because Terraform version is 0.11, however, since we are following same convention in other branch(TF0.12) i have change it to v1.7.0-11